eclipse-collections-code-generator/src/main/resources/test/bag/mutable/abstractMutablePrimitiveBagTestCase.stg

```diff
@@ -83,6 +84,13 @@ public abstract class AbstractMutable<name>BagTestCase extends AbstractMutable<n
         Assert.assertEquals(<name>HashBag.newBagWith(<["2", "2", "3", "3", "3"]:(literal.(type))(); separator=", ">), bag.selectDuplicates());
     }
 
+    @Test
+    public void selectUnique()
+    {
+        Mutable<name>Bag bag = this.newWith(<["1", "2", "2", "3", "3", "3"]:(literal.(type))(); separator=", ">);
```

Comment:
Similar to abstractImmutablePrimitiveBagTestCase, can you please add this test case to mutable side:

```
Mutable<name>Bag bag2 = this.newWith(<["1", "2", "2", "3", "3", "3", "4", "5", "5", "6"]:(literal.(type))(); separator=", ">);
Assert.assertEquals(<name>Sets.mutable.of(<["1", "4", "6"]:(literal.(type))(); separator=", ">), bag2.selectUnique());
```